larq_compute_engine/core/packbits.h

```diff
@@ -6,11 +6,15 @@
 #include <cstring>
 #include <limits>
 
+#include "larq_compute_engine/core/macros.h"
```

Comment:
This was already discussed, but the `macros.h` file can no be entirely removed I think.